### PR TITLE
Add messenger=persist doc

### PR DESCRIPTION
**Pull request by vincentchalamon**
Cf. https://github.com/api-platform/core/pull/3617

### CODE REVIEW

File: core/messenger.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ We use the `status` attribute to configure API Platform to return a [202 Accepte
 It indicates that the request has been received and will be treated later, without giving an immediate return to the client.
 Finally, the `output` attribute is set to `false`, so the HTTP response that will be generated by API Platform will be empty, and the [serialization process](serialization.md) will be skipped.
 
+**Note:** when using `messenger=true` ApiResource attribute in a Doctrine entity, the Doctrine DataPersister is not called. You must use the `messenger="persist"` ApiResource attribute.
+**Note:** when using `messenger="input"` ApiResource attribute in a Doctrine entity, the Doctrine DataPersister is not called. You must use an array containing `persist` and `input` values.
```
Comment (soyuka) [inline core/messenger.md:74]: should'nt we add an example that includes using `input` and  `persist` like [this one](https://github.com/api-platform/core/pull/3617/files#diff-904ce4ab2a1ce137bd5ecd11a4efe612)?

Comment (vincentchalamon) [reply core/messenger.md:74]: Yes, sure, in which format: yaml? xml? annotation?

File: core/messenger.md [hunk 2]
```diff
@@ -70,8 +70,24 @@ We use the `status` attribute to configure API Platform to return a [202 Accepte
...
 Finally, the `output` attribute is set to `false`, so the HTTP response that will be generated by API Platform will be empty, and the [serialization process](serialization.md) will be skipped.
 
-**Note:** using `messenger=true` or `messenger="input"` triggers a dedicated Messenger data persister. Therefore, any other built-in data persisters won't be called (e.g. Doctrine data persister).
-If you want to send a message after saving an object (like a Doctrine entity), prefer [creating a custom data persister](https://api-platform.com/docs/core/data-persisters/#creating-a-custom-data-persister) to save your object and [dispatch your own message manually](https://symfony.com/doc/current/components/messenger.html#bus).
+**Note:** when using `messenger=true` ApiResource attribute in a Doctrine entity, the Doctrine DataPersister is not called. You must use the `messenger="persist"` ApiResource attribute.
+
+**Note:** when using `messenger="input"` ApiResource attribute in a Doctrine entity, the Doctrine DataPersister is not called. You must use an array containing `persist` and `input` values.
```
Comment (soyuka) [inline core/messenger.md:75]:
```suggestion
**Note:** when using `messenger="input"` ApiResource attribute in a Doctrine entity, the Doctrine DataPersister is not called. You must use an array containing `persist` and `input` if you want it to be called, for example:
```

**Comment by soyuka** [comment]
Thanks!

**Pull request merged**